runtime/gc_glue_java/ConfigurationDelegate.hpp

```diff
@@ -344,7 +345,18 @@ class MM_ConfigurationDelegate
 	{
 		_maximumDefaultNumberOfGCThreads = maxGCThreads;
 	}
-
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	void checkPointGCThreadCountWarningCheck(MM_EnvironmentBase *env)
+	{
+		MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
+		if (!extensions->userSpecifiedParameters._checkpointGCThreads._wasSpecified) {
+			extensions->checkpointGCthreadCount = OMR_MIN(extensions->checkpointGCthreadCount, extensions->gcThreadCount);
+		} else if (extensions->checkpointGCthreadCount > extensions->gcThreadCount) {
+			PORT_ACCESS_FROM_ENVIRONMENT(env);
+			j9nls_printf(PORTLIB, J9NLS_WARNING, J9NLS_GC_THREAD_VALUE_MUST_BE_ABOVE_WARN, env->getExtensions()->checkpointGCthreadCount);
```

Comment:
I think we can not reuse this message: it says something is wrong with `-Xgcthreads` option. However I believe in this case we need warning about `-XX:CheckpointGCThreads`. Is it right? If so, we need different new NLS message.

Reply:
Yeah I was going to, this is more like a placeholder right now to see if it works as we expect :)

Reply:
For the nls message translation, is there a specific translation engine or something to refer?

Reply:
You need to modify `j9modron.nls` only.